src/gunicorn_config.py

```diff
@@ -218,6 +218,21 @@ def worker_abort(worker):
     worker.log.info("worker received SIGABRT signal")
 
 
+def sampler(fields):
+    request_path = fields.get("request.path")
+    response_code = fields.get("response.status_code")
+
+    # never sample errors
+    if response_code and response_code >= 500:
```

Comment:
would it be better python to instead say:
```python
    if response_code is not None and response_code >= 500:
```
?

Reply:
Eh, I think it's fine.